std/stdio.d

```diff
@@ -725,7 +725,8 @@ $(D rawRead) always reads in binary mode on Windows.
     {
         import std.exception : enforce, errnoEnforce;
 
-        enforce(buffer.length, "rawRead must take a non-empty buffer");
+        if (!buffer.length)
+            enforce(false, "rawRead must take a non-empty buffer");
```

Comment:
Actually it's even more awkward than that, msg can't be a runtime string, it has to be a compile-time string that generates a string at runtime.